### PR TITLE
Add kubelet module fields

### PR DESCRIPTION
**Pull request by exekias**
Add all missing fields for `kubelet` module.

Related to #3965

### CODE REVIEW

File: metricbeat/module/kubelet/container/_meta/fields.yml [hunk 1]
```diff
@@ -1,9 +1,139 @@
...
+      type: group
+      description: >
+        Logs info
+      fields:
+        - name: available
+          type: group
+          fields:
+            - name: bytes
+              type: long
+              description: >
```
Comment (ruflin) [inline metricbeat/module/kubelet/container/_meta/fields.yml]: you can add here `formate: bytes` so the Kibana index pattern will contain it and will directly make sense of the unit.

File: metricbeat/module/kubelet/container/_meta/fields.yml [hunk 2]
```diff
@@ -1,9 +1,139 @@
...
+                Logs total capacity in bytes
+        - name: used
+          type: group
+          fields:
+            - name: bytes
```
Comment (ruflin) [inline metricbeat/module/kubelet/container/_meta/fields.yml]: See commetn above. Applies to all bytes fields.

**Comment by exekias** [comment]
I added unit tests, as you say, mistakes are still possible, let's see if we can improve integration tests later on

File: metricbeat/module/kubelet/container/_meta/fields.yml [hunk 3]
```diff
@@ -1,9 +1,139 @@
...
+              type: bytes
+              description: >
+                Logs total capacity in bytes
+        - name: used
+          type: group
+          fields:
+            - name: bytes
+              type: bytes
```
Comment (ruflin) [inline metricbeat/module/kubelet/container/_meta/fields.yml]: You have to add:

```
type: long
format: bytes
```
In ES it is still just a long, only the kibana index pattern knows about bytes. Same for all others.

Comment (exekias) [inline metricbeat/module/kubelet/container/_meta/fields.yml]: Ah sorry, that makes much more sense, should be ok now :innocent:

**Pull request merged**